lifecycle-operator/controllers/common/evaluationhandler.go

```diff
@@ -20,8 +20,13 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 )
 
+type EvaluationHandlerInterface interface {
```

Comment:
Please, add a [moq](https://github.com/matryer/moq) annotation like the following: https://github.com/keptn/lifecycle-toolkit/blob/e13b9be3217fdd5bf3a646dd3d6ba49438cbb9e6/metrics-operator/cmd/config/config.go#L9C1-L9C1
Please, also generate the files with the mocks :)